src/main/java/frc/robot/Robot.java

```diff
@@ -29,7 +29,7 @@
 
   /* Setup each subsystem */
   public static DriveTrain m_drive = new DriveTrain();
-
+  public static Hatch m_hatchIntake = new Hatch();
```

Comment:
the naming is slightly odd here. i suggest either changing the variable to m_hatch or the subsystem to HatchIntake

```suggestion
  public static Hatch m_hatch= new Hatch();
```